### PR TITLE
[DO_NOT_MERGE] Added integration tests

### PR DESCRIPTION
**Pull request by jhellar**
## Motivation

Integration tests for auth.

### CODE REVIEW

**Comment by wtrocki** [comment]
@jhellar Could we get label for this as we have done for IOS SDK. 
Really cool feature!

File: tests/test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    const other = webviews.find(webview => webview !== current.value);
+    await client.context(other);
+  }
+  return { contexts, windows };
+}
+
+describe('Test for ' + opts.desiredCapabilities.platformName, function() {
+  this.timeout(0);
+
+  before(async function() {
```
Comment (josemigallas) [inline tests/test.js]: Let's use fat arrow functions please.

File: tests/test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const wdio = require('webdriverio');
```
Comment (josemigallas) [inline tests/test.js]: Any reason not to use typescript in tests?

**Comment by wtrocki** [comment]
@jhellar What is the story here?

**Comment by jhellar** [comment]
@wtrocki we decided to run these tests nightly [1] instead. They will be stored in separate repository. Will close this PR now.

[1] https://issues.jboss.org/browse/AEROGEAR-2737

**Pull request closed without merging**